src/services/refactors/extractSymbol.ts

```diff
@@ -299,7 +299,7 @@ namespace ts.refactor.extractSymbol {
             return { errors: [createFileDiagnostic(sourceFile, span.start, length, Messages.cannotExtractRange)] };
         }
 
-        if (isJSDoc(start)) {

```

Comment:
wow, a node test called `isJSDoc` that only checks if the node is a jsdoc *comment* seems very misleading... maybe we should rename it to `isJSDocComment`

Reply:
https://github.com/microsoft/TypeScript/pull/44416

Reply:
(That's not precisely the same thing, but is definitely related)